src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateVersions = [];
+  let stClone = { ...state };
```

Comment:
better to come up with more descriptive naming, it is now unclear what `stClone` means